### ISSUE TITLE
`@keyframes zoomIn` There is a compatibility problem. It will flash under some IOS systems. 

### ISSUE THREAD

**Issue by hymhub** [animate-css/animate.css#1526]
### Describe The Bug

`@keyframes zoomIn` There is a compatibility problem. It will flash under some IOS systems. It is recommended to change it to :
```
@keyframes zoomIn {
  from {
    opacity: 0;
    -webkit-transform: scale3d(0.3, 0.3, 0.3);
    transform: scale3d(0.3, 0.3, 0.3);
  }

  50% {
    opacity: 1;
  }

  100% {
    -webkit-transform: scale3d(1, 1, 1);
    transform: scale3d(1, 1, 1);
  }
}
```

**Comment by eltonmesquita** [comment]
Hey @hymhub. Thanks for reporting. Could you please share more details on this? Devices tested? A reduced case?

**Comment by knufri** [comment]
> ### Describe The Bug
> `@keyframes zoomIn` There is a compatibility problem. It will flash under some IOS systems. It is recommended to change it to :
> 
> ```
> @keyframes zoomIn {
>   from {
>     opacity: 0;
>     -webkit-transform: scale3d(0.3, 0.3, 0.3);
>     transform: scale3d(0.3, 0.3, 0.3);
>   }
> 
>   50% {
>     opacity: 1;
>   }
> 
>   100% {
>     -webkit-transform: scale3d(1, 1, 1);
>     transform: scale3d(1, 1, 1);
>   }
> }
> ```
> 
> ### Steps To Reproduce
> _No response_
> 
> ### Expected Behavior
> _No response_
> 
> ### Screenshots
> _No response_
> 
> ### Desktop
> _No response_
> 
> ### Smartphone
> _No response_
> 
> ### Additional Context
> _No response_

Thanks @hymhub this solved my problem on Safari 15.5 on Mac OS

**Comment by hymhub** [comment]
> 嘿@hymhub. 感谢您的报告。你能分享更多关于这方面的细节吗？测试设备？减少的情况？

This was discovered by using the company's test machine in my previous work. I didn't pay attention to the specific model. Now I can't find the test machine. I can't provide detailed data. I'm very sorry.
